packages/in_app_purchase/ios/Classes/FLTSKProductRequestHandler.m

```diff
@@ -0,0 +1,144 @@
+//
+//  FLTSKProductRequestWrapper.m
+//  in_app_purchase
+//
+//  Created by Chris Yang on 1/10/19.
```

Comment:
I don't think we want to include these generated notices in the obj c files, but I'm not totally sure.

/cc @amirh

Reply:
Yeah we don't usually add these.